src/Microsoft.Dnx.Tooling/Restore/RestoreOperations.cs

```diff
@@ -27,42 +27,39 @@ public RestoreOperations(IReport report)
             sw.Start();
 
             var dependencies = new List<LibraryDependency>();
-            if (context.RuntimeSpecs != null && !string.IsNullOrEmpty(context.RuntimeName))
-            {
-                // HACK(davidfowl): This is making runtime.json support package redirects
 
-                // Look up any additional dependencies for this package
-                foreach (var runtimeSpec in context.RuntimeSpecs)
```

Comment:
This loop appears to have been the problem. This might affect NuGet since we pulled this code from there...